### PR TITLE
Feature places

### PR DESCRIPTION
**Pull request by aaronrchz**
Improved  showcase app
implemented  recomendarías feature.
Fixed some bugs.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,25 +1,65 @@
...
+  void goToDirectionsScreen(){
+    previousScreen = currentScreen;
+    setState(() {
+      currentScreen = DirectionsScreen(openAiApiKey: widget.openAiApiKey, googleDirectionsApiKey: widget.googleDirectionsApiKey, onBack: goToPrevious,);
+    });
+  }
+  void goToNearbyDirectionsScreen() {
+    previousScreen = currentScreen;
+    setState(() {
+      currentScreen = RequestDNearbyPlacesScreen(googleDirectionsApiKey: widget.googleDirectionsApiKey, openAiApiKey: widget.openAiApiKey, func: goToRecommendedPlaceScreen ,onBack: goToPrevious,);
```
Comment (davidscmx) [inline lib/main.dart]: @aaronrchz Please format these lines using the Dart formatter for vscode/cursor

File: lib/componets/llm/supported_lenguages.dart
Comment (davidscmx) [inline lib/componets/llm/supported_lenguages.dart:1]: @aaronrchz Fix type in file name

File: lib/main.dart [hunk 2]
```diff
@@ -1,25 +1,65 @@
...
+  runApp(HumanDirectionsApp());
 }
 
-class MyApp extends StatelessWidget {
-  MyApp({super.key});
+class HumanDirectionsApp extends StatefulWidget {
+  HumanDirectionsApp({super.key});
   final String openAiApiKey = dotenv.env['OPENAI_API_KEY'] ?? 'NO SUCH KEY';
   final String googleDirectionsApiKey =
       dotenv.env['GOOGLE_DIRECTIOS_API_KEY'] ?? 'NO SUCH KEY';
```
Comment (davidscmx) [inline lib/main.dart:17]: @aaronrchz Typo

File: lib/screens/home_screen.dart [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                style: TextStyle(
+                  fontSize: 24.0,
+                  color: Colors.white,
+                  decoration: TextDecoration.none,
+                ),
+                textAlign: TextAlign.center,
+              ),
+              const SizedBox(height: 20),
+              const Text(
+                  'Human directions, es un paquete que intenta mejorar las indicaciones de navegación GPS, usando inteligencia artificial para reorganizar dichas indicaciones a un vocabulario más familiar y entendible para cualquier persona.',
```
Comment (davidscmx) [inline lib/screens/home_screen.dart]: @aaronrchz We are going to need to change to english as the default language :)

File: README.md
Comment (davidscmx) [inline README.md:1]: @aaronrchz Eventually the goal is to use this as a library. So please look into how to modify the package to turn this into a library and add some an example about usage.

File: lib/main.dart (hunk 1 above)
Comment (aaronrchz) [reply lib/main.dart]: Resolved in last commit [0b9264e]

File: lib/componets/llm/supported_lenguages.dart
Comment (aaronrchz) [reply lib/componets/llm/supported_lenguages.dart:1]: Resolved in last commit [0b9264e]

File: lib/main.dart (hunk 2 above)
Comment (aaronrchz) [reply lib/main.dart:17]: Resolved in last commit [0b9264e]

File: lib/screens/home_screen.dart (hunk 3 above)
Comment (aaronrchz) [reply lib/screens/home_screen.dart]: Resolved in last commit [0b9264e].
The Human directions module had always the default language as English, however the showcase was indeed in Spanish, so the showcase app language was changed to English on the last commit.

File: README.md
Comment (aaronrchz) [reply README.md:1]: This will be worked on a new branch.

File: lib/componets/places/places.dart
Comment (davidscmx) [inline lib/componets/places/places.dart:1]: typo in folder name: "componets"

**Review by davidscmx — COMMENTED** [review]
I have no more comments. We can merge this

File: lib/componets/places/places.dart
Comment (aaronrchz) [reply lib/componets/places/places.dart:1]: Resolved in commit [cf996b2]

**Pull request merged**